src/github.com/getlantern/lantern-ui/app/js/services.js

```diff
@@ -56,97 +56,40 @@ angular.module('app.services', [])
     }
 
     var fnList = {
-      'GeoLookup': function(data) {
-        console.log('Got GeoLookup information: ', data);
-        if (data && data.Location) {
-            model.location = {};
-            model.location.lon = data.Location.Longitude;
-            model.location.lat = data.Location.Latitude;
-            model.location.resolved = true;
+      'settings': function(settings) {
+        console.log('Got Lantern default settings: ', settings);
+        if (settings && settings.version) {
+            // configure settings
+            // set default client to get-mode
+            model.settings = {};
+            model.settings.mode = 'get';
+            model.settings.version = settings.version + " (" + settings.revision_date + ")";
         }
-      },
-      'Settings': function(data) {
-        if (data.Settings) {
-          var settings = data.Settings;
-          console.log('Got Lantern default settings: ', settings);
-          if (settings && settings.Version) {
-              // configure settings
-              // set default client to get-mode
-              model.settings = {};
-              model.settings.mode = 'get';
-              model.settings.version = settings.Version + " (" + settings.RevisionDate + ")";
-          }
-
-          if (settings.AutoReport) {
-              model.settings.autoReport = true;
-                  $rootScope.trackPageView();
-          }
-
-          if (settings.AutoLaunch) {
-              model.settings.autoLaunch = true;
-          }
-
-          if (settings.ProxyAll) {
-              model.settings.proxyAll = true;
-          }
 
-          if (settings.SystemProxy) {
-              model.settings.systemProxy = true;
-          }
+        if (settings.auto_report) {
+          model.settings.auto_report = true;
+          $rootScope.trackPageView();
```

Comment:
Yeah ideally we'd put it somewhere else, but we do ideally only want to report them when there's an actual pageview and not on every click. That said, it's just recording a page view so won't impact unique users, which is the more important stat, so I wouldn't worry too much about it.

It looks like your version is the same as the previous except the name change though right @xiam?